ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/NuGetPushTask.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+using YamlDotNet.Serialization;
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class NuGetPushTask : GithubTask
+    {
+        [YamlMember(Order = 1)]
+        public string Run = "dotnet nuget push";
+
+        internal string SearchPath { get; set; }
+
```

Comment:
If there's no attribute on top of these properties - we should remove the empty lines between them